docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -318,7 +318,7 @@ def numeric_version_from(tag)
 
       def registry_hostname
         return dependency.requirements.first[:source][:registry] if dependency.requirements.first[:source][:registry]
-        return credentials_finder.fetch_base_registry if credentials_finder.replaces_base?
+        return credentials_finder.base_registry if credentials_finder.base_registry
 
         "registry.hub.docker.com"
```

Comment:
I guess this line is redundant now.

Reply:
You mean the line 323?

Reply:
Yep, since the base_registry returns registry.hub.docker.com.

Reply:
Yes, you are right, we can remove that line.